pkgs/build-support/dotnet/nuget-to-nix/nuget-to-nix.sh

```diff
@@ -3,44 +3,80 @@
 set -euo pipefail
 
 export PATH="@binPath@"
+# used for glob ordering of package names
+export LC_ALL=C
 
 if [ $# -eq 0 ]; then
-  >&2 echo "Usage: $0 <packages directory> [path to file with a list of excluded packages] > deps.nix"
+  >&2 echo "Usage: $0 <packages directory> [path to excluded package source] > deps.nix"
   exit 1
 fi
 
 pkgs=$1
-exclusions="${2:-/dev/null}"
-tmpfile=$(mktemp /tmp/nuget-to-nix.XXXXXX)
-trap "rm -f ${tmpfile}" EXIT
+tmp=$(realpath "$(mktemp -td nuget-to-nix.XXXXXX)")
+trap "rm -r $tmp" EXIT
```

Comment:
Should use single quotes so that `$tmp` gets expanded upon `EXIT` instead of immidately.
```suggestion
trap 'rm -r $tmp' EXIT
```